dolphinscheduler-worker/src/main/java/org/apache/dolphinscheduler/server/worker/processor/TaskExecuteProcessor.java

```diff
@@ -161,7 +163,7 @@ public void process(Channel channel, Command command) {
         }
 
         // submit task to manager
-        boolean offer = workerManager.offer(new TaskExecuteThread(taskExecutionContext, taskCallbackService, alertClientService, taskPluginManager));
+        boolean offer = workerManager.offer(new TaskExecuteThread(taskExecutionContext, taskCallbackService, alertClientService, taskPluginManager, StorageOperateManager.getStorageOperate(ResUploadType.HDFS)));
```

Comment:
I think we support both `HDFS` and `S3` resources, but in this case, we use HDFS default and without the `S3` type. Should we support `S3` too?

Reply:
Yes, the currently created resources do not have a clear file storage mark to which they belong. After confirming with the test, the current default is HDFS. I think this can be optimized in the future. @zhongjiajie

Reply:
we should add an issue to connect this ASAP